src/observable/dom/AjaxObservable.ts

```diff
@@ -396,7 +396,7 @@ export class AjaxResponse {
       case 'json':
         if ('response' in xhr) {
           //IE does not support json as responseType, parse it internally
-          this.response = xhr.responseType ? xhr.response : JSON.parse(xhr.response || xhr.responseText || '');
+          this.response = xhr.responseType ? xhr.response : JSON.parse(xhr.response || xhr.responseText || 'null');
         } else {
           this.response = JSON.parse(xhr.responseText || '');
```

Comment:
isn't this `JSON.parse(xhr.responseText || '');` also need to be updated to `'null`' for fallback string? this isn't directly related to IE, but still seems possible unexpected failure cases.